README.md

```diff
@@ -198,6 +198,51 @@ class ClientIndexMessageHandler
 end
 ```
 
+#### Handling Messages with Resque
+
+If you use [Resque][resque], and you wish to handle messages in a resque job, you can use `Pwwka::QueueResqueJobHandler`, which is an adapter between the
+standard `handle!` method provided by pwwka and your Resque job.
+
+1. First, modify your `Gemfile` or otherwise arrange to include `pwwka/queue_resque_job_handler`:
```

Comment:
This kinda sucks, but I didn't want users to have to bring in resque if they aren't using it.

The other alternative would be something like:

```ruby
# in pwwka.rb
require 'pwwka/queue_resque_job_handler'

# in pwwka/queue_resque_job_handler.rb
begin
  require 'resque'
  module Pwwka
    class QueueResqueJobHandler
      # ...
    end
  end
rescue => LoadError
  puts "Install resque to use QueueResqueJobHandler" # or maybe something nicer instead of puts
end
```
This seems needlessly hacky to me, but I dunno.

Any thoughts?